Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/util/AccessType.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * #%L
+ * This file is part of "Apromore Core".
+ * %%
+ * Copyright (C) 2018 - 2020 Apromore Pty Ltd.
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-3.0.html>.
+ * #L%
+ */
+package org.apromore.util;
+
+/**
+ * Access types in ACL, which can map to flags in Group_[artifact] POJOs.
+ *
+ |        | has_read | has_write | has_ownership |
+ |--------|----------|-----------|---------------|
+ | Viewer | 1        | 0         | 0             |
+ | Editor | 1        | 1         | 0             |
+ | Owner  | 1        | 1         | 1             |
+ *
+ */
+public enum AccessType {
+
+    NONE(0),
+    VIEWER(1),
+    EDITOR(2),
+    OWNER(3);
+
+    private final Integer flagId;
+
+    AccessType(Integer flagId) {
+        this.flagId = flagId;
+    }
+
+    public Integer getFlogId() {
+        return flagId;
+    }
+}
```

Comment:
Misspelling: getFlagId.

Reply:
flagId can be an internal variable name, but the method name getFlagId() doesn't show its meaning. It can be given a better name according its intention here: e.g. getAccessTypeCode? I would suggest to rename the flagId variable to be a better name like accessTypeCode.